conkit/core/mappings.py

```diff
@@ -0,0 +1,102 @@
+from enum import Enum, unique
+
+
+class AminoAcidMapping(Enum):
+    """Amino acid mapping to encode an alignment"""
+    A = 1
+    B = 21
+    C = 2
+    D = 3
+    E = 4
+    F = 5
+    G = 6
+    H = 7
+    I = 8
+    J = 21
+    K = 9
+    L = 10
+    M = 11
+    N = 12
+    O = 21
+    P = 13
+    Q = 14
+    R = 15
+    S = 16
+    T = 17
+    U = 21
+    V = 18
+    W = 19
+    X = 21
+    Y = 20
+    Z = 21
+
```

Comment:
I presume there is a logic to this, but why are B, J, O, U, X and Z all set to 21?

Reply:
Oh, just remembered what the letters were representing and how many amino acids there are